docs/topics/proc_deploying-an-application-to-standalone-rhel.adoc

```diff
@@ -0,0 +1,107 @@
+// This is a parameterized module. Parameters used:
+//
+// parameter-version-runtime-plugin: runtime-specific name for the runtime version property variable in the POM file.
+// parameter-artifact-name-runtime-plugin: runtime plugin name in the POM file.
+// parameter-group-name-runtime-plugin: runtime plugin Maven artifact group name.
+// parameter-runtime-name: Runtime specific atribute for substituting the approrpiate expanded rumtime name.
+// parameter-maven-plugin-command-run: the runtime-specific Maven plugin command used to run the application.
+// parameter-link-appdev-guide: link to the procedure providing a complete example of the appropriate runtime-specific POM file configuration.
+//  context: used in anchor IDs to conflicts due to duplicate IDs.
+//
+// Rationale: This procedure is the same for 2 or more runtimes.
+[id='deploying-an-application-to-standalone-rhel_{context}']
+= Deploying an application to standalone Red Hat Enterprise Linux
+
+Deploy your {parameter-runtime-name} application to Red Hat Enterprise Linux.
+
+.Prerequisites
+
+* A Maven-based {parameter-runtime-name} application
+* The Red Hat Maven repository link:{parameter-link-appdev-guide}[specified] in the `pom.xml` of your project.
+* RHEL 7 installed.
+* OpenJDK 8 installed.
+* Maven 3.5 or later installed.
+
+
+.Procedure
+
+. Navigate to the directory with your application project:
++
+[source,bash,subs="attributes+",options="nowrap"]
+----
+$ cd myApp
+----
+
+. Ensure that the {parameter-runtime-name} Maven plugin is configured correctly in the `pom.xml` of your project:
```

Comment:
Originally I wanted to remove this section because of its redundant information.
How I see the doc now:

1. Create app
...
1.2. Define pom.xml
1.3. Create a class
...
3. Deploying an application to standalone
...
3.2 Ensure if in defined pom.xml from 1. step is correctly set ${runtime}-maven-plugin

Reply:
@inoxx03 if you want to keep it like that so please add this step to openshift with fabric8-maven-plugin and ${runtime}-maven-plugin subsection too. WDYT @Ladicek and @geoand?